src/filesystem/FileSystem.js

```diff
@@ -261,7 +261,8 @@ define(function (require, exports, module) {
     FileSystem.prototype._watchOrUnwatchEntry = function (entry, watchedRoot, callback, shouldWatch) {
         var impl = this._impl,
             recursiveWatch = impl.recursiveWatch,
-            commandName = shouldWatch ? "watchPath" : "unwatchPath";
+            commandName = shouldWatch ? "watchPath" : "unwatchPath",
+            ignored = watchedRoot.globFilter;
```

Comment:
`ignored` in chokidar allows you to use multiple things that match `anymatch` which are `Array|String|RegExp|Function`. RegExp and function we have to exclude, we can't transport them over to a domain. So just Array and String remains, it can be either glob string or an array of glob strings. Hence I wouldn't use generic ignore name but something that specifies that glob is expected.